### PR TITLE
Introduce Carthage compatibility

### PR DESCRIPTION
**Pull request by b-onc**
Let's see what CI says

### CODE REVIEW

**Comment by nuno-vieira** [comment]
@b-onc Don't forget to update the README.md that we now support Carthage 🙂

**Comment by b-onc** [comment]
@nuno-vieira README doesn't include installation instructions so i updated `overview.md`

File: docusaurus/docs/iOS/basics/overview.md [hunk 1]
```diff
@@ -108,8 +108,18 @@ The above command will generate the **MyProject.xcworkspace** file automatically
...
+github "GetStream/stream-chat-swift" ~> 4.0
+```
+
+and run:
+
+```bash
+carthage update --use-xcframeworks --platform iOS
+```
+
+go into your project, in the General settings tab, in the Frameworks, Libraries, and Embedded Content section, drag and drop each XCFramework you use from the Carthage/Build folder on disk.
```
Comment (nuno-vieira) [inline docusaurus/docs/iOS/basics/overview.md:125]: Maybe worth mentioning that if you only want the LLC, you don't need to link the StreamChatUI

Comment (b-onc) [reply docusaurus/docs/iOS/basics/overview.md:125]: "...drag and drop each XCFramework *you use*..." is enough IMHO. If a developer is only using LLC, they'd know not to include UI framework

**Review by nuno-vieira — APPROVED** [review]
LGTM

**Pull request merged**